### PR TITLE
Added contact us info

### PR DESCRIPTION
**Pull request by shovanch**
Added contact us information
Issue: #28

### CODE REVIEW

File: home/templates/home.html [hunk 1]
```diff
@@ -29,6 +29,10 @@
     <ul class="nav navbar-nav">
       <li class="active"><a href="#">Home</a></li>
       <li><a href="#">Resources</a></li>
+      <li><a href="http://bit.ly/pclubfb">Facebook</a></li>
```
Comment (divyam3897) [inline home/templates/home.html]: A seperate div has to be added for contact us with all the icons. You can use font awesome or similar icons for the same.

Comment (shovanch) [reply home/templates/home.html]: Should it go into navbar?

Comment (divyam3897) [reply home/templates/home.html]: yes, it should have a contact option in navbar and should have it's own section where we just provide the contact us things.

**Comment by sagar-kalra** [comment]
Use bootstrap icons for adding links.

**Comment by divyam3897** [comment]
Contact details added in https://github.com/pclubuiet/website/commit/44e7fdc0bfc83b6f32e9c013a08dda949fcf2f9a Thanks

**Pull request closed without merging**